include/gatekeeper_acl.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Gatekeeper - DoS protection system.
+ * Copyright (C) 2016 Digirati LTDA.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef _GATEKEEPER_ACL_H_
+#define _GATEKEEPER_ACL_H_
+
+#include "gatekeeper_config.h"
+#include "gatekeeper_net.h"
+
+/* TODO Add fields and rules to match IPv6 BGP packets for the CPS block. */
+
+/* Types of packets returned by Gatekeeper IPv6 ACL. */
+enum ipv6_acl_rule_type {
+	GATEKEEPER_IPV6_NO_MATCH,
+	GATEKEEPER_IPV6_ACL_ND,
+	GATEKEEPER_IPV6_ACL_MAX,
+};
```

Comment:
Why are these constants need? Why can't we have callbacks that are registered dynamically?